bzt/modules/influxdb_reporter.py

```diff
@@ -0,0 +1,378 @@
+"""
+Influxdb reporter for Taurus stats
+
+Copyright 2015 BlazeMeter Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import sys
+
+import bzt
+import time
+
+from influxdb import InfluxDBClient
+
+from bzt.engine import Reporter, Singletone
+from bzt.modules.aggregator import DataPoint, KPISet, AggregatorListener, ResultsProvider
+from bzt.utils import iteritems
+from influxdb.exceptions import InfluxDBServerError, InfluxDBClientError
+
+
+class InfluxdbStatusReporter(Reporter, AggregatorListener, Singletone):
+
+    METRIC_COUNT = "count"
+    METRIC_COUNT_ERROR = "countError"
+    METRIC_HIT = "hit"
+    METRIC_AVG = "avg"
+    METRIC_SENT_BYTES = "sb"
+    METRIC_RECEIVED_BYTES = "rb"
+    METRIC_PCT_PREFIX = "pct"
+    METRIC_MEAN_ACTIVE_THREADS = "meanAT"
+    TAG_TRANSACTION = "transaction"
+    TAG_TRANSACTION_INTERNAL = "internal"  # used with internal metrics (e.g. active threads, started threads ...)
```

Comment:
unused property TAG_TRANSACTION_INTERNAL

Reply:
Hy @matus-gazo 
Thanks for your feedback. I removed it.

Reply:
@RaphC Thank you! I will merge it and create release 1.16.21 with these changes.